code/modules/projectiles/guns/projectile/constructable/gunsmithing.dm

```diff
@@ -743,6 +743,80 @@
 			update_wheelchair_assembly()
 //CANNON END///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+//WIND-UP BEGIN////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/obj/item/weapon/windup_assembly
+	name = "wind-up assembly"
+	desc = "A spring in a shoe and all the associated possibilities."
+	icon = 'icons/obj/weaponsmithing.dmi'
+	icon_state = "windup_assembly_0"
+	var/windupstate = 0
+	var/list/windup_adhesive = list(
+		/obj/item/gum,
+		/obj/item/taperoll/police,
+		/obj/item/taperoll/engineering,
+		/obj/item/taperoll/atmos,
+		/obj/item/taperoll/syndie,
+		/obj/item/device/label_roll,
+		/obj/item/weapon/glue,
+		/obj/item/weapon/bonegel,
+		/obj/item/stack/package_wrap,
+		/obj/item/stack/medical/bruise_pack/bandaid,
+		/obj/item/weapon/reagent_containers/food/snacks/caramelburger
```

Comment:
wat

Reply:
Anything you can use as tape or glue. An adhesive.

Reply:
no, I was commenting on using a random burger as adhesive

Reply:
Caramel is sticky

Reply:
Why not other caramel foods? Why not other sticky foods? Just seems weird and unimmersive, like if you could us spaghetti to "tie it on"

Reply:
I couldn't think of any other significantly sticky foods. I just added it as a goofy little secret. Like 3 years from now some dude gets a smile because he finds out you can make them with caramel burgers.

Reply:
You can but it has a resist time of 1 second to represent that it's a bad idea, since this doesn't fall apart based on the components you use, it seems pretty questionable